src/metabase/lib/card.cljc

```diff
@@ -170,7 +170,7 @@
   "If `card` itself has a source card that is a Model, return that Model's columns."
   [metadata-providerable :- ::lib.schema.metadata/metadata-providerable
    card                  :- ::lib.schema.metadata/card]
-  (when-let [card-query (some->> (:dataset-query card) (lib.query/query metadata-providerable))]
+  (when-let [card-query (some->> (:dataset-query card) not-empty (lib.query/query metadata-providerable))]
```

Comment:
Do not explode on Cards that have empty (`{}`) queries... this is the column default value and we have a lot of tests that do this  (too many for me to want to fix)